test/functionalTest/viewDiffs.sh

```diff
@@ -0,0 +1,54 @@
+# Copyright 2015 Telefonica Investigacion y Desarrollo, S.A.U
```

Comment:
I have created this script inspired in the way `git difftool` works. It is similar to the current diff tool feature in testHarness.sh  but in a "post-mortem" way. I mean, after running testHarness.sh with a set of failing tests, running the following:

`CB_DIFF_TOOL=meld ./viewDiff.sh`

will go failing test after failing test asking if the diff want to be shown.